README.md

```diff
@@ -246,22 +250,22 @@ To see detailed general output to monitor progress, use the flag `-v` or
 `--verbose`.
 
 ```shell
-pipenv run python3 gatorgrouper/utils/gatorgrouper.py --verbose
+pipenv run python3 gatorgrouper-cli.py --verbose
```

Comment:
Is it `-` or `_`?

Reply:
It should be `_` and it should now be fixed